homeassistant/components/twinkly/__init__.py

```diff
@@ -0,0 +1,19 @@
+"""The twinkly component."""
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.helpers.typing import HomeAssistantType
+
+
+async def async_setup(hass: HomeAssistantType, config: dict):
+    """Set up the twinkly integration."""
+
+    return True
+
+
+async def async_setup_entry(hass: HomeAssistantType, config_entry: ConfigEntry):
+    """Set up entries from config flow."""
+
+    hass.async_create_task(
+        hass.config_entries.async_forward_entry_setup(config_entry, "light")
+    )
+    return True
```

Comment:
It's better to also add unload entry function to support unload and reload from UI

Reply:
Didn't knew about it, I've added it ... it was a easy one 😎